src/Parser/Mage/Frost/Modules/Features/Abilities.js

```diff
@@ -20,6 +20,7 @@ class Abilities extends CoreAbilities {
         cooldown: 45,
         enabled: combatant.hasTalent(SPELLS.EBONBOLT_TALENT.id),
         castEfficiency: {
+          disabled: combatant.hasTalent(SPELLS.GLACIAL_SPIKE_TALENT.id) ? true : false,
```

Comment:
Leave a comment (like the one in the PR) explaining why cast effic should be disabled in this case